src/common/components/github_tools.py

```
@@ -98,6 +101,36 @@ async def _arun(self, pr_number: int, body: str):
         return "comment created successfully"
 
 
+GET_ISSUE_BODY_PROMPT = """
+This tool is a wrapper for the GitHub API, useful when you want to get the body (description) of a Issue. **VERY IMPORTANT**: The issue number must be specified as an integer, not a float.
+"""
+
+
+class GetIssueBody(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)